packages/core/src/lib/FetchUtils.ts

```diff
@@ -15,14 +15,18 @@
  */
 
 import * as fs from 'fs';
-import {fetch as fetchh2, Response as FetchH2Response} from 'fetch-h2';
+import {context, Response as FetchH2Response} from 'fetch-h2';
 import nodefetch, {Response as NodeFetchResponse} from 'node-fetch';
 
 export type FetchEngine = 'node-fetch' | 'fetch-h2';
 const DEFAULT_FETCH_ENGINE: FetchEngine = 'fetch-h2';
 
 export type NodeFetchOrFetchH2Response = FetchH2Response | NodeFetchResponse;
 
+const userAgent = 'Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/81.0';
+const fetchh2Ctx = context({userAgent: userAgent, overwriteUserAgent: true});
+const fetchh2 = fetchh2Ctx.fetch;
```

Comment:
Are `fetchh2Ctx` and `fetchh2` used anywhere?

Reply:
Yes, line 41 is where `fetchh2` is used and `fetchh2Ctx` is used to generate the fetch call. Maybe we should collapse line 27 and 28 together?

Reply:
Oh right, you were previously importing something as `fetchh2` but now you're creating a variable of the same name. Sorry I missed that